package.json

```diff
@@ -119,7 +118,7 @@
     "@types/request-promise": "^4.1.48",
     "@types/semver": "^7.3.9",
     "@types/uuid": "^8.3.4",
-    "@types/ws": "^8.5.3",
+    "@types/ws": "^8.5.10",
```

Comment:
This PR does not aim to bump versions of deps, especially dev deps.

Reply:
`"@types/ws` was removed